Project/Send-MailKitMessage.csproj

```diff
@@ -4,9 +4,9 @@
     <AssemblyName>Send_MailKitMessage</AssemblyName>
     <Authors>Eric Austin</Authors>
     <PackageId>Send-MailKitMessage</PackageId>
-    <Version>3.2.0-preview1</Version>
+    <Version>3.2.0-preview2</Version>
     <AssemblyVersion>3.0.0</AssemblyVersion>
-    <FileVersion>3.2.0.11</FileVersion>
+    <FileVersion>3.2.0.12</FileVersion>
```

Comment:
These changes can be removed, these values get set during the NuGet publish process